README.md

```diff
@@ -262,7 +264,158 @@ export default NuxtAuthHandler({
 })
 ```
 
-Note that there's way more options inside the `nextAuth.options` object, see [here](https://next-auth.js.org/configuration/options#options) for all available options.
+##### Example with a custom Strapi JWT provider 
+
+Here's what a full config can look like, wee allow authentication via:
+- Strapi v4 JWTs with the `CredentialsProvider`
+
+Note that the below implementation of the credentials provider stores the Strapi JWT in the profile field of the user. The user object will be wrapped inside the session. Sidenote: the session has it's own JWT. Sending the Nuxt-Auth session JWT token as bearer token will not work, you need to pass the token that is originally returned by Strapi to get authorised accessing any auth enabled resources on your Strapi Server.
+
+For development, you can stay with the [Quick Start](#quick-start)-configuration.
+
+There's three places to configure `nuxt-auth` to work with Strapi:
+- `STRAPI_BASE_URL` in `.env`: Add the Strapi environment variable to your .env file.
+- [`runtimeConfig.STRAPI_BASE_URL`-key in `nuxt.config.ts`](#nuxtconfigts): Add the Strapi base url env variable. 
+- [`runtimeConfig.STRAPI_API_TOKEN_SECRET`-key in `nuxt.config.ts`](#nuxtconfigts): Optional: Allows server-to-server/machine-to-machine/headless usage through Strapi API Tokens. 
+- [`auth`-key in `nuxt.config.ts`](#nuxtconfigts): Configure the module itself, e.g., where the auth-endpoints are, what origin the app is deployed to, ...
+- [NuxtAuthHandler](#nuxtauthhandler): Configure the authentication behavior, e.g., what authentication providers to use
+
+
+For a production deployment, you will have to at least set the:
+- `origin` inside the `nuxt.config.ts` config (equivalent to `NEXTAUTH_URL` environment variable),
+- `secret` inside the `NuxtAuthHandler` config (equivalent to `NEXTAUTH_SECRET` environment variable)
```

Comment:
this is duplicated and not specific for the strapi example,. right? so please remove it an dbe strapi specific